drivers/gles2/rasterizer_storage_gles2.cpp

```diff
@@ -1506,6 +1510,11 @@ void RasterizerStorageGLES2::_update_shader(Shader *p_shader) const {
 			shaders.actions_scene.usage_flag_pointers["DEPTH_TEXTURE"] = &p_shader->spatial.uses_depth_texture;
 			shaders.actions_scene.usage_flag_pointers["TIME"] = &p_shader->spatial.uses_time;
 
+			// use of any of these BUILTINS indicate the need for transformed tangents
+			// this is needed to know when to transform tangents in software skinning
```

Comment:
Nitpick, but we're in the process of unifying the comment style to full sentence case (with final dots).